openreview/tools.py

```diff
@@ -65,6 +64,28 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     else:
         return profile
 
+def create_profile_ref(client, existing_profile, new_profile_content, source_id, debug=False):
+    # creates new profile with only the new content
+    profile = openreview.Profile(referent=existing_profile.id,
+                                 invitation=existing_profile.invitation,
+                                 signatures=[source_id],
+                                 writers=[source_id],
+                                 content=new_profile_content)
+    try:
+        profile = client.update_profile(profile)
+        if debug:
+            # test if fails to get original
+            db_profile = client.get_profile(id)
+    except Exception as e:
+        print "Exception {}".format(e)
+        print id
+    if debug:
+        super_group = client.get_group('OpenReview.net')
+        if len(super_group.members) > 1:
+            print super_group.members
```

Comment:
you should call `print` as a function in all these cases, because we're trying to keep things "bilingual" between python 3 and 2. one of the ways we do that is by importing the print_function from __future__ (see line 3).